components/list/list.css

```diff
@@ -0,0 +1,33 @@
+@import url("https://fonts.googleapis.com/css2?family=Mukta:wght@400;700&display=swap");
+* {
+  font-family: Mukta;
+}
+
+.list-container {
+  display: flex;
+  font-size: 1.2rem;
+  justify-content: space-evenly;
+}
+
+.unordered {
+  list-style: disc;
+}
+
+.ordered {
+  list-style: upper-roman;
```

Comment:
you should create diff classes for diff list styles, 
reason: as here you are giving to an ordered class a list-style; upper-raman, and if you import your Component lib somewhere and give the same class name, it will give roman numbers to list unnecessary. 
So you can have another class for roman styling